android/src/main/kotlin/com/fullstory/fullstory_flutter/FullstoryFlutterPlugin.kt

```diff
@@ -45,11 +45,9 @@ class FullstoryFlutterPlugin() : FlutterPlugin, MethodCallHandler {
         val message = call.argument<String>("message")
         val level = when (levelB) {
           0 -> FS.LogLevel.LOG // A.K.A. Verbose
-          1 -> FS.LogLevel.DEBUG
-          2 -> FS.LogLevel.INFO
-          3 -> FS.LogLevel.WARN
-          4 -> FS.LogLevel.ERROR
-          5 -> FS.LogLevel.ERROR // Assert on iOS; not actually enabled at the moment because it's a reserved keyword in dart.
+          1 -> FS.LogLevel.INFO
+          2 -> FS.LogLevel.WARN
+          3 -> FS.LogLevel.ERROR
           else -> FS.LogLevel.INFO // default. Alternatively we could error here.
```

Comment:
Oop, I just realized I made it default to INFO on Android for unrecognized log levels, but error on iOS. We should probably make those match. (I'd lean towards an error on both.)

Do you want to fix that for me in this PR? (I'm with you on generally preferring smaller ones, but I think this could make sense here.)

Reply:
Made the switch over to have android error as well (I agree it makes sense to lump that in here). Good catch!